refarch-eai/pom.xml

```diff
@@ -32,6 +32,7 @@
         <sonar-maven-plugin.version>3.9.0.2155</sonar-maven-plugin.version>
         <pmd-maven-plugin.version>3.23.0</pmd-maven-plugin.version>
         <spotbugs-maven-plugin.version>4.8.6.2</spotbugs-maven-plugin.version>
+        <spotbugs-annotations.version>4.8.6</spotbugs-annotations.version>
```

Comment:
Must this match the version of spotbugs-maven-plugin? (at least until X.X.X patch level). If yes I'd add a comment after <spotbugs-annotations.version> which indicates it has to match